### PR TITLE
fix the serialize bug when delete a Qdrant point

### PR DESCRIPTION
**Pull request by EashShow**
### Motivation and Context

fixed the bug #970 

### Contribution Checklist

- [x] The code builds clean without any errors or warnings
- [x] The PR follows SK Contribution Guidelines (https://github.com/microsoft/semantic-kernel/blob/main/CONTRIBUTING.md)
- [x] The code follows the .NET coding conventions (https://learn.microsoft.com/dotnet/csharp/fundamentals/coding-style/coding-conventions) verified with `dotnet format`
- [x] All unit tests pass, and I have added new tests where possible
- [x] I didn't break anyone :smile:

### CODE REVIEW

**Comment by EashShow** [comment]
@microsoft-github-policy-service agree

File: dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantMemoryStore.cs [hunk 1]
```diff
@@ -376,6 +398,12 @@ public async Task RemoveWithPointIdBatchAsync(string collectionName, IEnumerable
         return (record.Item1, record.Item2);
     }
 
+    /// <inheritdoc/>
```
Comment (lemillermicrosoft) [inline dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantMemoryStore.cs:401]: This isn't part of the `IMemoryStore` interface.

**Review by lemillermicrosoft — CHANGES_REQUESTED** [review]
This seems to be doing more than fixing  bug, and it's hard to see where any bug fix is happening. The UT added doesn't do any deletion of points. It seems like it's more about adding a new method ListPoints that should likely be part of the IMemoryStore interface and deserves a full proposal.

**Comment by Poltuu** [comment]
@EashShow would you mind splitting this PR in two parts, the part fixing the deserialization bug, and the other one adding a method to the `IQdrantVectorDbClient` interface ? Your fix would be dearly appreciated here 🙏

**Comment by EashShow** [comment]
> @EashShow would you mind splitting this PR in two parts, the part fixing the deserialization bug, and the other one adding a method to the `IQdrantVectorDbClient` interface ? Your fix would be dearly appreciated here 🙏

Yes, no problem

File: dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantMemoryStore.cs (hunk 1 above)
Comment (EashShow) [reply dotnet/src/Connectors/Connectors.Memory.Qdrant/QdrantMemoryStore.cs:401]: This is about the #1057

**Comment by rliberoff** [comment]
When can we expect a new version of the NuGet packages?

This is something that we are needing.

**Comment by shawncal** [comment]
The original bug #970 has already been fixed, and what remains here is mostly documentation and additional functionality. @EashShow closing the PR, but if you feel the other changes are valuable, please re-submit with description of added features and new tests.

**Pull request closed without merging**